### PR TITLE
Added alias ls='ls -F'

### LINKED ISSUE #1206: Recommended to add the option `-F` in `alias ls`
The `ls` option `-F` is useful and commonly used. `alias ls`  in `oh-my-zsh` also includes the option `-F`.
```bash
# ls, the common ones I use a lot shortened for rapid fire usage
alias l='ls -lFh'     #size,show type,human readable
alias la='ls -lAFh'   #long list,show almost all,show type,human readable
alias lr='ls -tRFh'   #sorted by date,recursive,show type,human readable
alias lt='ls -ltFh'   #long list,sorted by date,show type,human readable
alias ll='ls -l'      #long list
alias ldot='ls -ld .*'
alias lS='ls -1FSsh'
alias lart='ls -1Fcart'
alias lrt='ls -1Fcrt'
```

### PR DESCRIPTION
**Pull request by noviicee**
## Description

Added the option -F in alias ls.

## Motivation and Context

closes #1206 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by noviicee** [comment]
Kindly let me know if any changes are required. I'll be happy to do so. This is my first PR :)

File: aliases/available/general.aliases.bash [hunk 1]
```diff
@@ -15,6 +15,7 @@ alias la='ls -AF'       # Compact view, show hidden
 alias ll='ls -al'
 alias l='ls -a'
 alias l1='ls -1'
+alias ls='ls -F'
```
Comment (davidpfarrell) [inline aliases/available/general.aliases.bash:18]: This overwrites the `ls` alias settings above (see lines 4-10)

File: aliases/available/msys2.aliases.bash [hunk 2]
```diff
@@ -13,3 +13,4 @@ test -n "$LS_COMMON" && alias ls="command ls $LS_COMMON"
 alias ll="ls -l"
 alias la="ls -a"
 alias lal="ll -a"
+alias ls="ls -F"
```
Comment (davidpfarrell) [inline aliases/available/msys2.aliases.bash:16]: This overwrites the `ls` alias settings above (see line 12).

If used, the `-F` should be made part of `LS_COMMON`

**Review by davidpfarrell — CHANGES_REQUESTED** [review]
Sending back for changes ...

**Comment by noviicee** [comment]
Okay. Will fix it by tomorrow 👍

**Comment by gaelicWizard** [comment]
I like `-F` and support including it here

**Comment by NoahGorny** [comment]
gentle ping @noviicee

**Comment by noviicee** [comment]
So sorry for the delay. Changing the alias name immediately.

**Review by davidpfarrell — APPROVED** [review]
Approved, thanks !

**Pull request merged**